stats.go

```diff
@@ -85,6 +85,8 @@ type GetHeaderStatsRecord struct {
 	ValidatorID               string        `json:"validator_id"`
 	Latency                   int64         `json:"latency"`
 	UserAgent                 string        `json:"user_agent"`
+	SlotUID                   string        `json:"slot_uid"`
```

Comment:
maybe in another PR, but I think we should add builder_pubkey as well here (Or the PR that formally updates the table on the infra side could add it, and then we add builder_pubkey here later)